crates/nu-cli/src/env/host.rs

```diff
@@ -126,8 +126,9 @@ impl Host for BasicHost {
     }
 
     fn width(&self) -> usize {
-        let (term_width, _) = term_size::dimensions().unwrap_or_else(|| (80, 20));
-        term_width
+        let (mut term_width, _) = term_size::dimensions().unwrap_or_else(|| (80, 20));
+        term_width -= 1;
+        std::cmp::max(term_width, 80)
```

Comment:
Why do you return `max` as well?

Reply:
because i want the max of term_width and 80 - i don't want the terminal limited to 80 columns wide. is that not correct?

Reply:
The `max` is what limits the terminal to 80 columns, and so I think it's not necessary. The `.unwrap_or_else` part, though, is for when termsize can't determine column width at all (e.g. on Wasm target), so that one is a default.